.vscode/settings.json

```diff
@@ -1,17 +1,29 @@
 {
-    "editor.tabSize": 2,
-    "editor.insertSpaces": true,
-    "editor.detectIndentation": false,
-    "typescript.tsdk": "node_modules/typescript/lib",
-    "docker.commands.runInteractive": [
-      {
-        "label": "Default run command",
-        "template": "docker run --rm -it ${exposedPorts} ${tag}"
-      },
-      {
-        "label": "Run docs container",
-        "template": "docker run --rm -it ${exposedPorts} -v ${workspaceFolder}/docs:/docs ${tag}",
-        "match": "squidfunk"
-      }
-    ]
+  "editor.tabSize": 2,
+  "editor.renderFinalNewline": false,
+  "files.insertFinalNewline": false,
+  "editor.insertSpaces": true,
+  "editor.detectIndentation": false,
+  "editor.guides.bracketPairs": true,
+  "editor.accessibilitySupport": "off",
+  "editor.inlineSuggest.enabled": true,
+  "editor.formatOnSave": true,
+  "[typescriptreact]": {
```

Comment:
Is this for the .tsx files that we get in the test SPFx projects? If so, then let's remove it, because we should treat these projects as read-only and not modify them in any way.